parm/atm/obs/config/sondes.yaml

```diff
@@ -1,74 +1,44 @@
 obs space:
   name: sondes
   obsdatain:
-    engine:
+    engine: 
       type: H5File
-      obsfile: $(OBS_DIR)/$(OBS_PREFIX)sondes.$(OBS_DATE).nc4
-    obsgrouping:
-      group variables: ["station_id"]
-      sort variable: "air_pressure"
-      sort order: "descending"
+      obsfile: sondes_obs_$(OBS_DATE).nc4
   obsdataout:
     engine:
       type: H5File
-      obsfile: $(DIAG_DIR)/diag_sondes_$(OBS_DATE).nc4
-  io pool:
-    max pool size: 1
-  simulated variables: [eastward_wind, northward_wind, air_temperature, specific_humidity, surface_pressure]
-get values:
-  interpolation method: $(INTERP_METHOD)
+      obsfile: sondes_diag_$(OBS_DATE).nc4
+  simulated variables: [surface_pressure, air_temperature, eastward_wind, northward_wind, specific_humidity]
+geovals:
+  filename: sondes_geoval_$(OBS_DATE).nc4
+vector ref: GsiHofXBc
+tolerance: 0.01
 obs operator:
   name: Composite
   components:
    - name: VertInterp
      variables:
      - name: air_temperature
-#    - name: virtual_temperature
-     - name: specific_humidity
      - name: eastward_wind
      - name: northward_wind
+     - name: specific_humidity
    - name: SfcPCorrected
-     variables:
-     - name: surface_pressure
      da_psfc_scheme: GSI
-linear obs operator:
-  name: Composite
-  components:
-   - name: VertInterp
-     variables:
-     - name: air_temperature
-     - name: specific_humidity
-     - name: eastward_wind
-     - name: northward_wind
-   - name: Identity
-     variables:
+     geovar_sfc_geomz: surface_geometric_height
+     geovar_geomz: geopotential_height
+     variables: 
      - name: surface_pressure
-#
+linear obs operator:
+  name: Identity
```

Comment:
I don't think linear obs operator should be `identity` for all ob types here

Reply:
Updated linear obs operator for all the obs variables used in the file.